app/src/main/java/app/revanced/manager/util/Util.kt

```diff
@@ -108,4 +118,50 @@ suspend fun <T> Flow<Iterable<T>>.collectEach(block: suspend (T) -> Unit) {
             block(it)
         }
     }
+}
+
+fun Int.formatNumber(): String {
+    return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+        NumberFormatter.with()
+            .notation(Notation.compactShort())
+            .decimal(NumberFormatter.DecimalSeparatorDisplay.ALWAYS)
+            .precision(Precision.fixedFraction(1))
+            .locale(Locale.getDefault())
+            .format(this)
+            .toString()
+    } else {
+        val compact = CompactDecimalFormat.getInstance(
+            Locale.getDefault(), CompactDecimalFormat.CompactStyle.SHORT
+        )
+        compact.maximumFractionDigits = 1
+        compact.format(this)
+    }
+}
+
+fun String.relativeTime(): String {
+    try {
+        val currentTime = ZonedDateTime.now(ZoneId.of("UTC"))
+        val inputDateTime = ZonedDateTime.parse(this)
+        val duration = Duration.between(inputDateTime, currentTime)
+
+        return when {
+            duration.toMinutes() < 1 -> "Just now"
+            duration.toMinutes() < 60 -> "${duration.toMinutes()}m ago"
+            duration.toHours() < 24 -> "${duration.toHours()}h ago"
+            duration.toDays() < 30 -> "${duration.toDays()}d ago"
+            else -> {
```

Comment:
This is not translatable

Reply:
This isn't a composable, how do I get translateable text here?